SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -91,15 +111,24 @@ Because LLVM integers use a two’s complement representation, this instruction
 @[simp_llvm]
 def sub? {w : Nat} (x y : BitVec w) : IntW w :=
   pure <| x - y
+structure SubParams where
+  nuw : Bool := false
+  nsw : Bool := false
```

Comment:
Given that `AddParams` is the same as `SubParams` is the same as `MulParams`, it makes sense to unify those into a single `OverflowFlags` structure